model_compression_toolkit/common/post_training_quantization.py

```diff
@@ -459,8 +462,7 @@ def _prepare_model_for_quantization(in_model: Any,
     # Shift Negative Activations
     ######################################
     if quant_config.shift_negative_activation_correction and \
-            quant_config.enable_activation_quantization and \
-            quant_config.activation_quantization_method is not QuantizationMethod.UNIFORM:
+            quant_config.enable_activation_quantization:
```

Comment:
Why has the condition here changed?

Reply:
Cause quant_config does not have activation_quantization_method  anymore.

Reply:
Still we need this check let talk about it